lib/internal/child_process.js

```diff
@@ -800,7 +800,7 @@ function _validateStdio(stdio, sync) {
       stdio = i < 3 ? 'pipe' : 'ignore';
     }
 
-    if (stdio === null || stdio === 'ignore') {
+    if (stdio === 'ignore') {
```

Comment:
In previous condition we're handle `stdio === null`, so this condition never will be executed with `stdio === null`